src/js/view/controls/controls.js

```diff
@@ -308,12 +331,11 @@ export default class Controls {
 
     userInactive() {
         clearTimeout(this.activeTimeout);
-        this.showing = false;
-        if (this.controlbar) {
-            this.controlbar.closeMenus({
-                type: 'userInactive'
-            });
+        if (this.settingsMenu.visible) {
```

Comment:
What happens if the volume tooltip menu is open? 

Also, if you're hovering over the control bar, the `user-inactive` flag does not get added after the timeout.
<img width="934" alt="screen shot 2017-08-16 at 1 20 42 am" src="https://user-images.githubusercontent.com/4603952/29348405-59b766f6-8221-11e7-9fec-77332a8867e7.png">